lib/normalizeResponse.ts

```diff
@@ -1,17 +1,17 @@
-function stringify (obj) {
+export const stringify = (obj: any) => {
   return typeof obj === 'object' ? JSON.stringify(obj) : obj
 }
 
-function normalizeResponse (response) {
+export const normalizeResponse = (response: any) => {
   const {
     isBase64Encoded = false,
     statusCode = 200,
     headers = {},
     multiValueHeaders = {},
     body,
-    html,
-    json,
-    xml
+    html = {},
+    json = {},
+    xml = {}
```

Comment:
Need to be careful with args in a few places, since these are assumed to be falsy. We could solve this by typing `response` with defs like `html?: string`.

Reply:
I have no idea how most of your data structures work tbh. I just tried to shut up the compiler sometimes 🙃

Reply:
Haha fair. Yeah we'll pry have some `any` and stuff scattered to start.